src/app/legacy/psammead/psammead-topic-tags/src/index.jsx

```diff
@@ -60,13 +60,11 @@ const SingleTopicTagItem = styled.div`
   }
 `;
 
-export const TopicTag = forwardRef(
-  ({ name, link, onClick = null, ...viewTracker }, ref) => (
-    <a href={link} onClick={onClick} ref={ref} {...viewTracker}>
-      {name}
-    </a>
-  ),
-);
+export const TopicTag = forwardRef(({ name, link, ...handlers }, ref) => (
```

Comment:
Could we rename this to something more descriptive such as `staticComponentTrackers` perhaps?

```suggestion
export const TopicTag = forwardRef(({ name, link, ...handlers }, ref) => (
```